unison.opam

```
@@ -1,25 +1,38 @@
+# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-maintainer: "juergen@hoetzel.info"
-authors: [
-  "Trevor Jim"
-  "Benjamin C. Pierce"
-  "Jérôme Vouillon"
-]
+version: "dev"
+synopsis: "File-synchronization tool for Unix and Windows"
+description: """
+Text based user interface for Unison.
+
+Unison is a file-synchronization tool for Unix and Windows.
+It allows two replicas of a collection of files and directories
+to be stored on different hosts (or different disks on the same host),
+modified separately, and then brought up to date by propagating
+the changes in each replica to the other.
+"""
+maintainer: ["juergen@hoetzel.info"]
+authors: ["Trevor Jim" "Benjamin C. Pierce" "Jérôme Vouillon"]
 license: "GPL-3.0-or-later"
 homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+doc: "https://github.com/bcpierce00/unison/wiki"
 bug-reports: "https://github.com/bcpierce00/unison/issues"
-dev-repo: "git://github.com/bcpierce00/unison.git"
-build: ["dune" "build" "-p" name "-j" jobs]
 depends: [
-  "ocaml" {build & >= "4.08"}
-  "ocamlfind" {build}
-  "dune" {build & >= "2.3"}
-  "lablgtk3" {build & >= "3.1.0"}
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.3"}
 ]
-synopsis: "File-synchronization tool for Unix and Windows"
-description: """
-Unison is a file-synchronization tool for Unix and Windows. It allows
-two replicas of a collection of files and directories to be stored on
-different hosts (or different disks on the same host), modified
-separately, and then brought up to date by propagating the changes in
-each replica to the other."""
+build: [
+  ["dune" "subst"] {pinned}
```

This is a bug in the opam file autogeneration fixed in dune 2.7 (see https://github.com/ocaml/dune/pull/3647). I would suggest to bump to at least `(lang dune 2.7)`.
```suggestion
  ["dune" "subst"] {dev}
```